tests/ssl-opt.sh

```diff
@@ -1460,6 +1460,7 @@ trap cleanup INT TERM HUP
 # - the expected parameters are selected
 #   ("signature_algorithm ext: 6" means SHA-512 (highest common hash))
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
+requires_ciphersuite_enabled TLS-ECDHE-RSA-WITH-CHACHA20-POLY1305-SHA256
```

Comment:
Optional: I know this PR isn't supposed to do everything, but if I may suggest a low-hanging fruit: these tests also only make sense if SHA-512 and Curve25519 are enabled.